test/mock-server/routers.ts

```diff
@@ -0,0 +1,226 @@
+import { Router, Status } from './deps.ts'
+import type { FormDataBody } from './deps.ts'
+
+import { generateToken } from './jwt.ts'
+import responses from './responses.ts'
+
+const applicantId = '742af521-2c8f-4b54-a8b7-01c962849d5b'
+
+/**
+ * This helper is to match against uploaded files
+ * instead of using switch-case
+ * because IE11 on Windows submit files in a weird format
+ * e.g: C:Windowsproxyd0bd7093-f21a-4579-8707-f7aabd4948c2upload4900467381210572080filepassport.jpg
+ */
+const matchUploadedFile = (
+  uploadedFileName: string,
+  matchFileNames: string[]
+): boolean => {
+  const regex = new RegExp(`(${matchFileNames.join('|')})$`)
+  return uploadedFileName.match(regex) != null
+}
```

Comment:
Do we still need to use this helper given we are not going to run tests on IE11?

Reply:
It's still a good piece of helper that we can define multiple files to match once. Plus, the validation is really simple (match the file name at the end of the file path), so it still serves well in general.